spec/presenters/changed_documents_pagination_presenter_spec.rb

```diff
@@ -19,137 +15,134 @@ def build_presenter_for_subtopic(total: 100, start: 0, page_size: 50, view_conte
     it "returns the document count divided by the per_page value" do
       presenter = build_presenter_for_subtopic(total: 200, page_size: 20)
 
-      assert_equal 10, presenter.total_pages
+      expect(presenter.total_pages).to eq(10)
     end
 
     it "rounds up to the next page" do
       presenter = build_presenter_for_subtopic(total: 201, page_size: 20)
 
-      assert_equal 11, presenter.total_pages
+      expect(presenter.total_pages).to eq(11)
     end
 
     it "returns zero pages when there are zero results" do
       presenter = build_presenter_for_subtopic(total: 0)
 
-      assert_equal 0, presenter.total_pages
+      expect(presenter.total_pages).to eq(0)
     end
   end
 
   describe "#next_page?" do
     it "is true if there is a subsequent page" do
       presenter = build_presenter_for_subtopic(start: 0, total: 100)
 
-      assert presenter.next_page?
+      expect(presenter).to be_next_page
```

Comment:
I've been manually changing these sorts of things to eg 
`expect(presenter.next_page).to be(true)`

Reply:
like you have in the next test actually - you must just have missed this one

Reply:
same comment for lines 52 and 58 below

Reply:
Ta. They're quite hard to see after a while 👍